resources/articles/adaptive_layer_height_enabled.md

```diff
@@ -0,0 +1,8 @@
+Adaptive layers calculates the optimal layer height based on the 3D model's surface features.
+The layer height will be reduced when there is a slope on that part of the model, and increased when the wall is completely vertical.
+
+![A hemisphere with adaptive layers disabled (150 micron layer height).](images/adaptive_layers_disabled.png)
+
+![A hemisphere with adaptive layers enabled (150 micron base layer height).](images/adaptive_layers_enabled.png)
+
+A more extensive blog post about this feature was published on the [Ultimaker website](https://ultimaker.com/en/blog/52520-print-detailed-objects-faster-using-adaptive-layers-in-ultimaker-cura).
```

Comment:
Almost all articles in this guide are also explained on the Ultimaker website, but for users it's more convenient if they can directly read it here. This is the whole reason why the guide can exist, really!